WatchDogEclipsePlugin/WatchDog/src/nl/tudelft/watchdog/logic/document/EditorWrapper.java

```diff
@@ -0,0 +1,23 @@
+package nl.tudelft.watchdog.logic.document;
+
+import nl.tudelft.watchdog.core.logic.document.EditorWrapperBase;
+
+import org.eclipse.ui.texteditor.ITextEditor;
+
+/**
+ * Wrapper class for Eclipse editor.
+ */
+public class EditorWrapper extends EditorWrapperBase {
```

Comment:
Maybe, I've just wanted to be consistent everywhere (WatchDogUtils/Base, Preferences/Base, EditorWrapper/Base).